LocalPackages/Waitlist/Sources/Waitlist/Network/ProductWaitlistRequest.swift

```diff
@@ -133,7 +133,7 @@ public class ProductWaitlistRequest: WaitlistRequest {
 
     private var endpoint: URL {
 #if DEBUG
-        return URL(string: "https://quackdev.duckduckgo.com/api/auth/waitlist/")!
+        return URL(string: "https://quack.duckduckgo.com/api/auth/waitlist/")!
```

Comment:
This change exists as the NetP backend doesn't support invite codes from quackdev.